### ISSUE TITLE
[Bug] Pressing 3 or 4 buttons at the same time causes the game to hitch

### ISSUE THREAD

**Issue by Kishidle** [Drewol/unnamed-sdvx-clone#309]
**Describe the bug**
Pressing 3 or 4 buttons at the same time causes the game to hitch, on any song, using an SVSE5 controller. More buttons = stronger hitch. Issue also exists if I use my mechanical keyboard,

**Expected behavior**
Causes the game to hitch. On my setup it usually runs at around 2k FPS and I see it drop to 1.5k for a bit each time I press 3 or 4 buttons. 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - GPU: GTX 1070
 - CPU: Ryzen 3700
 - Version: 0.4.0

**Comment by Drewol** [comment]
Are you using a skin? because I cannot replicate this with everything as default.

**Comment by Kishidle** [comment]
Nope, running default skin. It's a freshly installed copy of 0.4.0.

**Comment by Kishidle** [comment]
Update: I managed to fix it. It's either 1) running it in administrator mode, or 2) running usc-game.exe in w7 compatibility mode.
